Gems/Multiplayer/Code/CMakeLists.txt

```diff
@@ -68,7 +156,43 @@ ly_add_target(
             Include
     BUILD_DEPENDENCIES
         PRIVATE
-            Gem::Multiplayer.Static
+            Gem::Multiplayer.Unified.Static
+    RUNTIME_DEPENDENCIES
+        Gem::CertificateManager
+)
+
+ly_add_target(
+    NAME Multiplayer.Client ${PAL_TRAIT_MONOLITHIC_DRIVEN_MODULE_TYPE}
+    NAMESPACE Gem
+    FILES_CMAKE
+        multiplayer_shared_files.cmake
+    INCLUDE_DIRECTORIES
+        PRIVATE
+            Source
+            .
+        PUBLIC
+            Include
+    BUILD_DEPENDENCIES
+        PRIVATE
+            Gem::Multiplayer.Client.Static
+    RUNTIME_DEPENDENCIES
+        Gem::CertificateManager
```

Comment:
do you need this `RUNTIME_DEPENDENCIES` entry if the static library already has it?

Reply:
The static library doesn't have it. Are you looking at the Multiplayer target? That's the Unified version of this target. I kept it Multiplayer instead of Multiplayer.Unified for compatibility purposes.